lib/ansible/modules/network/junos/junos_l3_interfaces.py

```diff
@@ -374,10 +374,10 @@
      of the parameters above.
   type: list
 commands:
-  description: The set of commands pushed to the remote device.
+  description: The set of commands in xml pushed to the remote device.
   returned: always
   type: list
-  sample: ['command 1', 'command 2', 'command 3']
+  sample: ['xml 1', 'xml 2', 'xml 3']
```

Comment:
This should probably have some real examples to clearly articulate what to expect form the module.

Reply:
you think so? sampling netconf xml might be a little overkill imho

Reply:
Hey Sam, not really. NetConf XMLs could be quite lenghty. If you're ok with his for now I'll merge this PR and apply this to L2 as well. Thanks!

Reply:
I'm just echoing @Qalthos here so approval should come from him.  The only other suggestion I have is to put an example up in github and link to it from the notes section.

Reply:
@Qalthos @samccann well, there's no samples on the commands on non-xml modules either. In any case if it's ok for you I'll post something on a follow-up patch.